src/main/java/weka/classifiers/trees/j48It/C45PruneableClassifierTreeIt.java

```diff
@@ -156,50 +166,29 @@ public void dumpTree(int depth, StringBuffer text) throws Exception {
 	  
 
 	  
-	  public void addToListWithPriority(ArrayList<Object[]> list, ArrayList<Object[]> listSons) {
-		    if (list.size() == 0) {
+	  public void addOrderedBySize(ArrayList<Object[]> list, ArrayList<Object[]> listSons) {
+		    if (list.size() == 0) 
+		    {
 		        list.add(0, listSons.get(0));
```

Comment:
Eta ez litzateke listSons-etik 0-garren objektua kendu beharko??? Bestela, hurrengo for-ean berriro gehituko litzateke, ezta? edo?

Reply:
Egia, hau gehitu dut lehenengo posizioa kentzeko:

if (list.size() > 0 && list.get(0)[0].equals(listSons.get(0)[0])) {
		  listSons.remove(0); // Remove the first element from the list listSons
}

Reply:
Nik hementxe jarriko nuke: listSons.remove(0);
Eta hurrengo if hori kendu